### PR TITLE
Add visualization postprocessors for thermal conductivity and diffusivity

### PR DESCRIPTION
**Pull request by anne-glerum**

### CODE REVIEW

File: source/postprocess/visualization/thermal_diffusivity.cc [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      ASPECT_REGISTER_VISUALIZATION_POSTPROCESSOR(ThermalDiffusivity,
+                                                  "thermal diffusivity",
+                                                  "A visualization output object that generates output "
+                                                  "for the thermal diffusivity $\\kappa$.")
```
Comment (bangerth) [inline source/postprocess/visualization/thermal_diffusivity.cc]: Can you state here that this equals $\frac{k}{c_p \rhp}$ where $k$ is the thermal conductivity? I am perpetually confused by which is which, and assume that others are too.

**Comment by bangerth** [comment]
Other than the one comment it looks good. Is there an easy test you could steal that is a small adaptation of similar postprocessors?

**Comment by gassmoeller** [comment]
I agree that these are missing postprocessors at the moment. Just a quick note: The thermal conductivity is already implemented in the 'material properties' postprocessor. I am advertising that one a bit, because it is much more efficient than all of these postprocessors that evaluate the material model, but output only a single material property. This might matter for material models, where the computation of the variables takes some time (e.g. because of iterations in the viscosity calculation). Nevertheless, for consistency we should add these as well.

**Comment by anne-glerum** [comment]
@gassmoeller Shall I add the thermal diffusivity to the 'material properties' postprocessor then too, to be completely consistent? :)

**Comment by gassmoeller** [comment]
If you want to, feel free :-), but I would merge this also without it. Could I also ask you to reduce the test by one refinement, or is this size required for something? That would make the output a bit smaller. Apart from that, everything looks ready for merge.

**Comment by anne-glerum** [comment]
When using the `material properties` pp with a property that is also listed as a separate pp, the double entry in the vtu file caused my Paraview to crash. It also doesn't make sense to compute the property twice, so I've added an assert to prevent the double listing.

**Comment by gassmoeller** [comment]
Nice work! Thanks.

**Comment by gassmoeller** [comment]
Would you mind writing an entry in changes.h? Afterwards I will merge, I promise :-).

**Pull request merged**